docs/CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ ServerPubKey = '...'
 These will eventually replace `TelemetryIngress.URL` and `TelemetryIngress.ServerPubKey`. Setting `TelemetryIngress.URL` and `TelemetryIngress.ServerPubKey` alongside `[[TelemetryIngress.Endpoints]]` will prevent the node from booting. Only one way of configuring telemetry endpoints is supported.
 - Added bridge_name label to `pipeline_tasks_total_finished` prometheus metric. This should make it easier to see directly what bridge was failing out from the CL NODE perspective.
 
+### Required Configuration Change
+
+- Starting in 2.7.0, chainlink nodes will no longer allow `Database.AllowSimplePasswords` to be set. Any TOML configuration that sets this field will prevent the node from booting, and will fail configuration validation.
```

Comment:
Wait, are we not allowing it to be _set at all_? That seems like overkill. We only need to disallow _using_ the feature. Setting the field to explicit `false` is not problematic, and blocking that could get in the way of some node ops.